src/programs/simulate/simulate.cpp

```diff
@@ -528,7 +532,7 @@ class SimulateApp : public MyApp {
         return (this->min_bFactor + pdb_bfactor * this->bFactor_scaling);
     }
 
-    inline float return_scattering_potential(corners& R, float* bPlusB, AtomType& atom_id) {
+    inline float return_scattering_potential(corners const& R, float* bPlusB, AtomType& atom_id) {
```

Comment:
We speed up the evaluation of the error function using MKL. This requires some batching.